### PR TITLE
[Bristol] Add mapfile and layer for parks

### PR DESCRIPTION
**Pull request by MorayMySoc**
* Initial map file
* Create with layer for park assets

### CODE REVIEW

File: layers/bristol.map [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    METADATA
+      "wfs_title"         "Bristol Parks"
+      "wfs_srs"           "EPSG:3857 EPSG:27700"
+      "gml_include_items" "SITE_NAME,SITE_CODE"
+      "gml_featureid"     "SITE_CODE"
+      "wfs_enable_request" "*"
+    END
+    TYPE POLYGON
+    STATUS ON
+    DATA 'bristol/CONTRACT_FEATURES_BySite'
```
Comment (dracos) [inline layers/bristol.map:29]: It doesn't really matter, I guess, but could this have had a better filename? :)

Comment (dracos) [reply layers/bristol.map:29]: The first entry in this file is a multipolygon with a blank site code and name?

**Review by dracos — APPROVED** [review]
Code looks fine (filename aside), one data question, not sure where that park is.

File: layers/bristol.map (hunk 1 above)
Comment (MorayMySoc) [reply layers/bristol.map:29]: I'm not really sure...is that something I need to handle? It doesn't sound normal, or is it?

Comment (dracos) [reply layers/bristol.map:29]: I've opened it in QGIS and it looks to be tiny, so probably just one to ignore (but is mistake in their data, we shouldn't have to handle it).

**Pull request merged**